### ISSUE TITLE
exception responses must be valid JSON

### ISSUE THREAD

**Issue by jgruber** [F5Networks/f5-declarative-onboarding#25]
Because the HTTP return status code is not sufficient to understand the state (you need the status constant and message attribute) of a declaration, the return body with errors MUST be valid JSON. The exception output often makes the response body non-valid JSON. 

Example 500 error body output which is not valid JSON:

```
{"code":500,"message":"java.net.ProtocolException: status:500, body:{\"code\":500,\"status\":\"ERROR\",\"message\":\"rollback failed\",\"errors\":[\"01070277:3: The requested vlan, vlangroup or tunnel (/Common/net_tap7ecab634_a4) was not found.\",\"01070277:3: The requested vlan, vlangroup or tunnel (/Common/net_tap7ecab634_a4) was not found.\",\"01070330:3: Static route gateway 1.1.1.88 is not directly connected via an interface.\",\"01070330:3: Static route gateway 1.1.1.88 is not directly connected via an interface.\"],\"declaration\":{\"$schema\":\"https://raw.githubusercontent.com/F5Networks/f5-declarative-onboarding/master/schema/base.schema.json\",\"async\":false,\"schemaVersion\":\"1.0.0\",\"Common\":{\"ntpServers\":{\"timezone\":\"UTC\",\"class\":\"NTP\",\"servers\":[\"0.pool.ntp.org\",\"1.pool.ntp.org\"]},\"licenseKey\":{\"licenseType\":\"regKey\",\"class\":\"License\",\"regKey\":\"UFRFP-YSMUH-DHDKG-LWORR-VVKWFOJ\",\"unitOfMeasure\":\"monthly\",\"overwrite\":false},\"configsync\":{\"configsyncIp\":\"1.1.1.110\",\"class\":\"ConfigSync\"},\"provisioningLevels\":{\"ltm\":\"nominal\",\"class\":\"Provision\",\"asm\":\"minimum\"},\"route_192_168_1_102_32\":{\"gw\":\"1.1.1.88\",\"mtu\":0,\"class\":\"Route\",\"network\":\"192.168.1.102/32\"},\"hostname\":\"testv14-do-additional.sample.openstack.f5se.com\",\"dnsServers\":{\"nameServers\":[\"8.8.8.8\"],\"search\":[\"sample.openstack.f5se.com\"],\"class\":\"DNS\"},\"net_tap58767ee9_ee\":{\"interfaces\":[{\"tagged\":false,\"name\":\"1.2\"}],\"tag\":4092,\"class\":\"VLAN\",\"mtu\":1450},\"tapec56899d-01_default_gw\":{\"gw\":\"192.168.80.1\",\"mtu\":0,\"class\":\"Route\",\"network\":\"default\"},\"selfip_tap7ecab634_a4\":{\"trafficGroup\":\"traffic-group-local-only\",\"vlan\":\"net_tap7ecab634_a4\",\"allowService\":\"all\",\"class\":\"SelfIp\",\"address\":\"1.1.1.110/24\"},\"net_tap7ecab634_a4\":{\"interfaces\":[{\"tagged\":false,\"name\":\"1.1\"}],\"tag\":4093,\"class\":\"VLAN\",\"mtu\":1450},\"selfip_tap58767ee9_ee\":{\"trafficGroup\":\"traffic-group-local-only\",\
```

**Comment by focrensh** [comment]
AT-215 for internal tracking

**Comment by seattlevine** [comment]
Resolved in 1.5.0
